src/csp_io.c

```diff
@@ -400,5 +408,7 @@ void csp_sendto_reply(const csp_packet_t * request_packet, csp_packet_t * reply_
 	if (opts & CSP_O_SAME) {
 		reply_packet->id.flags = request_packet->id.flags;
 	}
-	csp_sendto(request_packet->id.pri, request_packet->id.src, request_packet->id.sport, request_packet->id.dport, opts, reply_packet);
+	uint16_t dst = request_packet->id.src;
```

Comment:
uint16_t dst variable not needed

Reply:
We actually need it, in case the same buffer is reused for reply and request